src/utils/exr-parser.worker.js

```diff
@@ -17,35 +17,34 @@
  *  };
  */
 
+
 let openEXRLoaded = false;
 let queuedJobs = [];
 let OpenEXR;
 
-self.addEventListener('message', (event) => {
+self.addEventListener('message', function (event) {
     if (!openEXRLoaded) {
         queuedJobs.push(event.data);
     } else {
         handleJob(event.data);
     }
 });
 
-
-const exrwrapPath = require('file-loader?name=exr-wrap.js!../exr-wrap/exr-wrap.js');
-const exrwrapWasmPath = require('file-loader?name=exr-wrap.wasm!../exr-wrap/exr-wrap.wasm');
-
-importScripts(exrwrapPath);
-
-EXR().then(function(Module) {
-	OpenEXR = Module;
-	openEXRLoaded = true;
-	while (queuedJobs.length > 0) {
-		const job = queuedJobs.shift();
-		if (job) {
-			handleJob(job);
-		}
-	}
+// The wasmBinary will be read in as a function to be instantiated in the exr-wrap.js
+const wasmBinary = require('../exr-wrap/exr-wrap.wasm');
+const EXR = require("../exr-wrap/exr-wrap.js");
+EXR({
+  instantiateWasm: (info, receiveInstance) => wasmBinary(info).then(instance => receiveInstance(instance.instance))
```

Comment:
@cbreak-black After your comments yesterday it helped me take a step back, I realized that surely there is a way to do this without modifying the exr-wrap.js which shouldn't be modified, as you pointed out.

And yes there is.  I hacked it into the exr-wrap.js, and if I hacked it in, there should be a way to call that exr-wrap.js with the wasm, and there is, here is the line to invoke it.  To my knowledge this isn't documented anywhere I could find, can only find it from beautifying the exr-wrap.js and looking how it actually is instantiating the wasm, then mimicking that on the EXR module itself, works beautifully.  

So the reason this is crucial, this is what allows the unified build to work without dealing with handling the wasm externally.  Previously if a project wanted to use jeri, they have to host wasm and have it load from `location.origin` which can be problematic.  Now, it's embedded within the jeri.js itself and dynamically loaded as a function with no external XMLHttpRequest calls.